### PR TITLE
BDD Remove Content Feature

### PR DESCRIPTION
**Pull request by pcardiga**
BDD scenarios for Remove Content feature - https://jira.ez.no/browse/EZP-24385

The current BDD scenarios aim to reflect the behaviour of removing a content in PlatformUI
The situations reflected below are not yet reflected in BDD scenarios

Todo:
- [ ] Implementation

For later follow up:
- Remove one object when the user is not allowed to.
- Remove one object that has multiple locations
- Remove multiple objects
- Untrash a trashed object

### CODE REVIEW

File: Features/CopyMoveDelete/RemoveContent.feature [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        Then I see a "Are you sure you want to send this content to trash?" message
+        And I see a "Confirm" button
+        And I see a "Cancel" button
+
+    ##
+    #Delete objects
+    ##
+    @javascript
+    Scenario: Remove one object and confirm the removal
+        Given an "News Flash" article exists
```
Comment (crevillo) [inline Features/CopyMoveDelete/RemoveContent.feature]: typo ```Given a```?

**Comment by pcardiga** [comment]
ping @bdunogier , @dpobel , @andrerom

File: Features/CopyMoveDelete/RemoveContent.feature [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    Scenario: Removing one object should redirect to it's parent location view
+        Given a "News" folder exists
+        And a "News child" article exists under "News" folder
+        When I remove the "News child" article
+        And I confirm the removal
+        Then the "News child" article is removed
+        Then I am on the "News" view form
+
+    @javascript
+    Scenario: Content tree is updated after the removal of an object
```
Comment (bdunogier) [inline Features/CopyMoveDelete/RemoveContent.feature]: This is already covered line 24 above (`Scenario: Remove one object and confirm the removal`)

File: Features/CopyMoveDelete/RemoveContent.feature [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        And the "News child" article is removed
+
+    @javascript
+    Scenario: Removing one object should redirect to it's parent location view
+        Given a "News" folder exists
+        And a "News child" article exists under "News" folder
+        When I remove the "News child" article
+        And I confirm the removal
+        Then the "News child" article is removed
+        Then I am on the "News" view form
```
Comment (bdunogier) [inline Features/CopyMoveDelete/RemoveContent.feature]: "view form" ?

**Comment by pcardiga** [comment]
Scenarios updated
ping @bdunogier, @dpobel, @andrerom

**Comment by pcardiga** [comment]
Scenario sentences updated accordingly dicussion occurred in [Copy BDD PR](https://github.com/ezsystems/PlatformUIBundle/pull/243/ "PR for the BDD Copy feature")

**Comment by pcardiga** [comment]
Scenario sentences updated again accordingly new dicussion occurred in [Copy BDD PR](https://github.com/ezsystems/PlatformUIBundle/pull/243/ "PR for the BDD Copy feature")

**Comment by bdunogier** [comment]
I'm not sure it is worth updating all of them. I know that many of the discussions do apply to them all, but I also think it takes a lot of your time, for very little benefit.

What do you think we focus on the copy one, until we get it right ? When we're done, I think it's best to postpone merging until you're done updating the move/remove scenarii, and we got started on review a little bit.

**Comment by andrerom** [comment]
+1, ref commet on https://github.com/ezsystems/PlatformUIBundle/pull/243#issuecomment-119960457

**Comment by bdunogier** [comment]
+1

**Pull request merged**